legend-engine-xts-persistence/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/src/main/java/org/finos/legend/engine/persistence/components/relational/executor/RelationalExecutor.java

```diff
@@ -135,7 +135,7 @@ public Dataset constructDatasetFromDatabase(Dataset dataset)
     @Override
     public void setSqlLogging(SqlLogging sqlLogging)
     {
-        this.sqlLogging = sqlLogging;
+        this.sqlLogging = SqlLogging.UNMASKED;
```

Comment:
Is this for testing ?

Reply:
Sure, removed - was used in testing and accidentally committed